src/components/AddItem.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import './AddItem.css';
+
+export default function AddItem({ onAdd }) {
+  const [itemName, setItemName] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    onAdd(itemName);
+    setItemName('');
+  }
+
+  return (
+    <form className="Add-item__form" onSubmit={handleSubmit}>
+      <label htmlFor="new__item" className="Add-item__label">
+        Your article:
+      </label>
+      <input
+        id="new__item"
+        onChange={event => setItemName(event.target.value)}
+        value={itemName}
+        role="searchbox"
+        type="search"
+        className="Add__input"
+        required
+      />
+      <button aria-label={'click to add item'} className="Add__Button">
```

Comment:
The `aria-label` here is a bit unnecessary since the text within your button is sufficient enough.
```suggestion
      <button className="Add__Button">
```